lib/commands/add.ts

```diff
@@ -88,12 +95,17 @@ command = {
 				cd14: config.project.theme
 			});
 
-			await command.addTemplate(argv.name, selectedTemplate);
+			await command.addTemplate(argv.name, selectedTemplate, argv.module);
 			await PackageManager.flushQueue(true);
 			PackageManager.ensureIgniteUISource(config.packagesInstalled, command.templateManager);
 		}
 	},
-	async addTemplate(name: string, template: Template): Promise<boolean> {
+	async addTemplate(name: string, template: Template, modulePath?: string): Promise<boolean> {
+		if (modulePath && !Util.directoryExists(path.join(process.cwd(), modulePath))) {
```

Comment:
@wnvko This shouldn't really work - modulePath is supposed to point to a file so a directory check should always fail, no?
Bonus - the path is relative to the structure inside the project (e.g. "src/app/modulePath") so you can't quite check it against the current dir. I guess we can add it as an option to `generateFiles` (last optional parameter) so templates can verify it with the correct path.